src/erlcloud_ses.erl

```diff
@@ -43,6 +43,19 @@
 -export([verify_email_identity/1, verify_email_identity/2]).
 -export([verify_domain_identity/1, verify_domain_identity/2]).
 
+-export([ create_custom_verification_email_template/6,
```

Comment:
style nit: can you please regroup the exports like the rest of the module? The intent is for the export order to mimic the API documentation.

Reply:
I grouped and ordered them alphabetically. Next checkin.